externs/shaka/text.js

```diff
@@ -246,6 +246,21 @@ shaka.extern.Cue = class {
      */
     this.backgroundColor;
 
+    /**
+     * The number of horizontal and vertical cells into which
+     * the Root Container Region area is divided
+     *
+     * @typedef {{ columns: number, rows: number }}
```

Comment:
This looked different from what we would normally use, so I checked.  It seems that with this annotation, the compiler doesn't enforce the type correctly.  What you want is `@type {{ columns: number, rows: number }}` instead.

A typedef in Closure Compiler is used for creating a new type, but not for assigning a type to a member.  I know the compiler can be arcane, so I'm sorry for the trouble.